### DISCUSSION TITLE
Teleport -> GKE CloudSQL Proxy Sidecar Fails

### DISCUSSION THREAD

**Discussion by soudaburger** [gravitational/teleport#22695] [Q&A]
When attempting to connect to a sidecar running `cloudsqlproxy`, Teleport wants to connect to the sidecar over TLS, but `cloudsqlproxy` explicitly does NOT support TLS as a connection parameter.  It wants cleartext.  Is it possible to connect teleport to `cloudsqlproxy` as a sidecar? How do we disable TLS for database connections? Is that even possible?

```bash
connection to server at "localhost" (127.0.0.1), port 63833 failed: server refused TLS connection
```

**Comment by webvictim** [comment]
Teleport's `db_service` does not support plain text connections to downstream SQL servers.
